### DISCUSSION TITLE
How to force a package to rebuild.

### DISCUSSION THREAD

**Discussion by v3ss0n** [pdm-project/pdm#3112] [General]
I have stalled `llamacpp-python` but i forgot to make it built using cuda. So i am trying to reinstall using cuda suppoirt by doing this:
` CMAKE_ARGS="-DGGML_CUDA=on" pdm add llama-cpp-python`

But the problem is PDM just reinstall already built one instead of rebuilding with GPU support. 
I tried removing .venv folder , but it just install already build one.

How to force rebuild with the env var set.

**Comment by frostming** [comment]
`pdm cache remove <pattern>` to remove the built wheel

you can see what are cached with `pdm cache list`
